src/rest/controllers/Heroes.ts

```diff
@@ -1,17 +1,23 @@
-import { heroesWithTypes } from '../../utils/fragments';
 import { Request, Response } from 'express';
+
+import { heroesWithTypes } from '../../utils/fragments';
 import { prisma } from '../../../generated/prisma-client';
 
 export async function getAllHeroes(req: Request, res: Response) {
   const { first, skip } = req.query;
-  const heroes = await prisma
+  const data = await prisma
     .heroes({
       first: parseInt(first),
       skip: parseInt(skip),
     })
     .$fragment(heroesWithTypes);
 
-  res.send(heroes);
+  const totalCount: number = await prisma
+    .heroesConnection()
+    .aggregate()
+    .count();
+
+  res.send({ data, total_count: totalCount });
```

Comment:
I'm not sure if response is in the same format like before the change. @qmixi are you sure/have you checked it?

Reply:
Response will be different. Instead of having response of Array, we will have Object with 2 properties, `data` and `total_count`.
It's okey, but we need to update Wiki with documentation,

Reply:
Yes, exactly as Fank wrote. The response will be different, but it's the best way to handle this issue  :)